apps/etterlatte-vilkaarsvurdering/src/main/kotlin/no/nav/etterlatte/vilkaarsvurdering/VilkaarsvurderingRoutes.kt

```diff
@@ -60,6 +60,14 @@ fun Route.vilkaarsvurdering(
             }
         }
 
+        get("/{$BEHANDLINGID_CALL_PARAMETER}/rettUtenTidsbegrensning") {
```

Comment:
litt mer fan av `-` i url'er -  `/rett-uten-tidsbegrensning` er mer lettlest synes jeg. Men mulig vi har brukt denne varianten andre steder allerede?

Reply:
Vi har alle mulige varianter ser det ut til. Jeg er enig med deg kjenner jeg etter å ha tygd det litt. Endrer da i samme slengen `migrertYrkesskadefordel` som også er innført for aldersoverganger. Så blir det i allefall konsekvent navngivning i vv, ref `oppdater-status` i samme fil :)